Code/C++/left_rotation.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+left_rotation(arr, len) alters the int array of length len to shift every element of the array one position to it's left.
```

Comment:
Add this on top of left_rotation(arr, len) function as its seocific to this function

here you provide description of problem statement.